l10n/en-US/chrome.properties

```
@@ -15,5 +15,6 @@
 # Chrome notification bar messages and buttons
 unsupported_feature=This PDF document might not be displayed correctly.
 unsupported_feature_forms=This PDF document contains forms. The filling of form fields is not supported.
+unsupported_feature_signatures=This PDF document contains digital signatures. The signatures validation is not supported.
```

While I'm obviously not a native English speaker, the second sentence sounds a bit weird to me. How about the following instead?
```suggestion
unsupported_feature_signatures=This PDF document contains digital signatures. Validation of signatures is not supported.
```